website/content/docs/install/quickstart.mdx

```diff
@@ -46,6 +46,9 @@ run a single-node development instance using the `nomad agent -dev` command
 
 [Installing Nomad][installing-binary]
 
+-> **Note:** If you are using Docker Desktop for Windows or MacOS, please check
```

Comment:
Could we add this same note to the [docker driver page](https://www.nomadproject.io/docs/drivers/docker)?

Reply:
Good call, added. Thanks!